packages/client/src/__tests__/integration/errors/executeRaw-alter-postgres/test.ts

```diff
@@ -11,7 +11,6 @@ beforeAll(async () => {
   process.env.TEST_POSTGRES_URI += '-execute-raw-alter'
   await tearDownPostgres(process.env.TEST_POSTGRES_URI!)
   await migrateDb({
-    connectionString: process.env.TEST_POSTGRES_URI!,
```

Comment:
Is this not necessary?

Reply:
`connectionString` was not used within `migrateDb` so I removed it everywhere.